cmd/bootstrap/cmd/snapshot.go

```diff
@@ -0,0 +1,107 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+
+	"github.com/onflow/flow-go-sdk/client"
+
+	model "github.com/onflow/flow-go/model/bootstrap"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/state/protocol/inmem"
+)
+
+var flagAccessAddress string
+var flagNodeID string
+
+// snapshotCmd represents the command to download the latest protocol snapshot to bootstrap from
+var snapshotCmd = &cobra.Command{
+	Use:   "snapshot",
+	Short: "Downloads the latest serialized protocol snapshot from an access node to be used to bootstrap from",
+	Long:  `Downloads the latest serialized protocol snapshot from an access node to be used to bootstrap from`,
+	Run:   snapshot,
+}
```

Comment:
This command should live as a subcommand in the `transit` script rather than here since it will be used by node operators.